earth_enterprise/src/fusion/portableglobe/servers/fileunpacker/util.py

```diff
@@ -19,7 +19,15 @@
 
 import os
 import pexpect
-import pexpect.fdpexpect
+
+""" Try loading fdpexpect for version 3.* and above.
```

Comment:
This would probably become a docstring for the `util` module, rather than just a comment. You may want to use `#`.